src/main/scala/com/wavesplatform/matcher/api/MatcherApiRoute.scala

```diff
@@ -132,7 +133,13 @@ case class MatcherApiRoute(assetPairBuilder: AssetPairBuilder,
   @Path("/settings")
   @ApiOperation(value = "Matcher Settings", notes = "Get matcher settings", httpMethod = "GET")
   def getSettings: Route = (path("settings") & get) {
-    complete(StatusCodes.OK -> Json.obj("priceAssets" -> matcherSettings.priceAssets))
+
+    complete(
+      StatusCodes.OK -> Json.obj(
+        "priceAssets" -> matcherSettings.priceAssets,
+        "order-fee"   -> matcherSettings.orderFee.getJson(minMatcherFee).value
```

Comment:
Looks like we have a different naming style. It'd better to avoid this in API. Sorry, but we need to use camelCase here (`orderFee`), because people already use `priceAssets`. And in `getJson` too.

Reply:
Done